comps/comps-foreman-plugins-rhel7.xml

```diff
@@ -152,6 +152,7 @@
       <!--
         Do not edit this section manually and use ./comps_doc.sh script
       -->
+      <packagereq type="default">rubygem-openscap_parser</packagereq>
```

Comment:
Looks like the script places this in a weird location. Can you move it above the comment in a sorted location?